CMakeLists.txt

```diff
@@ -549,45 +549,6 @@ set(ZIG_STAGE2_SOURCES
     src/Value.zig
     src/Zcu.zig
     src/Zcu/PerThread.zig
-    src/arch/aarch64/CodeGen.zig
-    src/arch/aarch64/Emit.zig
-    src/arch/aarch64/Mir.zig
-    src/arch/aarch64/abi.zig
-    src/arch/aarch64/bits.zig
-    src/arch/arm/CodeGen.zig
-    src/arch/arm/Emit.zig
-    src/arch/arm/Mir.zig
-    src/arch/arm/abi.zig
-    src/arch/arm/bits.zig
-    src/arch/powerpc/CodeGen.zig
-    src/arch/riscv64/abi.zig
-    src/arch/riscv64/bits.zig
-    src/arch/riscv64/CodeGen.zig
-    src/arch/riscv64/Emit.zig
-    src/arch/riscv64/encoding.zig
-    src/arch/riscv64/Lower.zig
-    src/arch/riscv64/Mir.zig
-    src/arch/riscv64/mnem.zig
-    src/arch/sparc64/CodeGen.zig
-    src/arch/sparc64/Emit.zig
-    src/arch/sparc64/Mir.zig
-    src/arch/sparc64/abi.zig
-    src/arch/sparc64/bits.zig
-    src/arch/wasm/CodeGen.zig
-    src/arch/wasm/Emit.zig
-    src/arch/wasm/Mir.zig
-    src/arch/wasm/abi.zig
-    src/arch/x86/bits.zig
-    src/arch/x86_64/CodeGen.zig
-    src/arch/x86_64/Disassembler.zig
-    src/arch/x86_64/Emit.zig
-    src/arch/x86_64/Encoding.zig
-    src/arch/x86_64/Lower.zig
-    src/arch/x86_64/Mir.zig
-    src/arch/x86_64/abi.zig
-    src/arch/x86_64/bits.zig
-    src/arch/x86_64/encoder.zig
-    src/arch/x86_64/encodings.zon
```

Comment:
* Did you mean to delete all of these entries?
* I think you forgot to actually delete the sparc64 backend.

Reply:
Yes

sparc64 is not abandoned and it's further along than the others

Reply:
Didn't you want to delete sparc64 when we previously discussed this topic? If so, what changed your mind?